static/app/components/replays/videoReplayer.tsx

```diff
@@ -114,6 +123,20 @@ export class VideoReplayer {
     this.playSegmentAtIndex(nextIndex);
   }
 
+  /**
+   * Create videos from a slice of _attachments, given the start and end index.
+   */
+  protected createVideoForRange({low, high}: {high: number; low: number}) {
```

Comment:
l: this is a bit subjective, so low on logaf

generally if I have a function with a small amount of parameters, where they are required, I would use positional arguments. in this case also, since it's a range, it'd be consistent with a lot of native js functions that accept 2 args for start/end.

counter point, I can never remember the parameters between array slice/splice and having named/object parameters is nice for being self documenting.